pkg/operator/replacement_starter.go

```diff
@@ -199,6 +202,7 @@ func CreateControllerInputFromControllerContext(ctx context.Context, controllerC
 		apiregistrationv1Client:      apiregistrationv1Client,
 		migrationClient:              migrationClient,
 		eventRecorder:                eventRecorder,
+		clock:                        clock.RealClock{},
```

Comment:
controllerContext has a clock on it.  please wire to that.

Reply:
oops, fixed